### PR TITLE
Fixed code review requests

### PR DESCRIPTION
**Pull request by SevlaMare**
Change the navbar logo for a image and put a link on this. 
Bottom grid now are links and have image preview.
Correct general styling.

### CODE REVIEW

**Review by pelzolga123 — CHANGES_REQUESTED** [review]
Project requires changes.

**Comment by pelzolga123** [comment]
## Hi Team!  
### Your project looks :+1: amazing!

* [x] Pull Request should have meaningful title and a short description ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_prs.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))

### Navigation bar
* [x] It should have NY Times image logo embedded in the center ([link to page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html?_r=0)) Find image with original logo.
* [x] The NY Times logo should be a link ([link to page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html?_r=0))

### Bottom part
* [x] Should contain links to other articles with image previews on the leftmost side ([link to page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html?_r=0))

### General styling
* [x] Next part will look better if you make it more like original:
image
Original:
image

### Please kindly fix the above issues and send another code review request.

If you have any questions or need help, you can message me on slack (@ Olga Pelts). *Happy coding*

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,4 @@
+README.md
```
Comment (pelzolga123) [inline .gitignore]: `README.md`, `.stickler.yml`, `stylelint.config.js`  files should not be in `.gitignore`

**Comment by uryelah** [comment]
### Pull Request under the review by Sarah Uryelah Silva Chamorro (TSE)

**Comment by uryelah** [comment]
Hello **Ssegawa Alvin Louis K Ssegawa** & **Kamilu Akanji Adewale**, I'm Sarah, your reviewer for this PR .

You did a good job completing the required changes from the last review, congrats! 🥇 

**Great job at:**
- using semantic HTML
- writing tidy CSS
- great README.md file

### Your project is approved! 🎊 🥅 💛 
### 
**You may close this Pull Request and merge the changes**

-------

**Optional changes**

### Touch-ups!

- [x]  [OPTIONAL] The 'Theory of Inflation' area overlaps the images and text a bit on wide screens. Try to touch it up a bit. 

[Hint: Use dev tools to see the site in different screen sizes.](https://developer.mozilla.org/en-US/docs/Tools/Responsive_Design_Mode)
 
- [x]  [OPTIONAL] Change the nav box-shadow to look closer to the original.
Your nav:
Screenshot_2020-02-21 Article - New York Times(2)
[Original site:](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html)
Screenshot_2020-02-21 Space Ripples Reveal Big Bang’s Smoking Gun

-------

**Keep being awesome devs and good luck with your next project!**

Cheers and Happy coding!👏👏👏

Code review by **Sarah Uryelah Silva Chamorro**

(Feel free to contact me on slack by the handle of  **sarah** if you have any additional question)

**Pull request merged**